_data/benefits.yml

```diff
@@ -1,12 +1,12 @@
 section: benefits
 title: Benefits
```

Comment:
Also let's add a title to the section. Something along the lines of "Benefits" or "What you gain"